src/features/wallet/WalletControlBar.tsx

```diff
@@ -47,7 +48,7 @@ export function WalletControlBar() {
 
         {numReady === 1 && (
           <SolidButton onClick={() => setIsSideBarOpen(true)} classes="px-2.5 py-1" color="white">
-            <div className="flex items-center justify-center">
+            <div className="flex w-36 items-center justify-center xs:w-auto">
```

Comment:
TODO, test sol wallet

Reply:
Tested with eth wallet, cosmos wallet, sol wallet and multiple wallets and everything seemed fine!